rpc/src/rpc.rs

```diff
@@ -154,6 +157,28 @@ pub struct JsonRpcConfig {
     pub disable_health_check: bool,
 }
 
+impl Default for JsonRpcConfig {
+    fn default() -> Self {
+        Self {
+            enable_rpc_transaction_history: Default::default(),
+            enable_extended_tx_metadata_storage: Default::default(),
```

Comment:
Huh, clippy or CI doesn't complain about all these `Default::default()`s?
I'd prefer these be specific to type.

Reply:
It does not, and yeah it's weird. I'll change to use specific types.